game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbem/EmailSenderEditorViewModel.java

```diff
@@ -16,6 +16,8 @@
 
 class EmailSenderEditorViewModel {
   static final String PROVIDER_DISABLED = "Disabled";
+  static final String GMAIL = "Gmail";
```

Comment:
These values probably should not have been hardcoded to begin with. We do have constants for these values already though:
```
EmailProviderPreset.GMAIL.getName();
```

Reply:
Updated.